api/bazel/repositories.bzl

```diff
@@ -179,30 +183,24 @@ go_proto_library(
     proto = ":rpc_status_protos_lib",
     visibility = ["//visibility:public"],
     deps = [
-      "@com_github_golang_protobuf//ptypes/any:go_default_library",
+        "@com_github_golang_protobuf//ptypes/any:go_default_library",
     ],
 )
 
 py_proto_library(
-     name = "rpc_status_protos_py",
-     srcs = [
-         "google/rpc/status.proto",
-     ],
-     include = ".",
-     default_runtime = "@com_google_protobuf//:protobuf_python",
-     protoc = "@com_google_protobuf//:protoc",
-     visibility = ["//visibility:public"],
-     deps = ["@com_google_protobuf//:protobuf_python"],
+    name = "rpc_status_protos_py",
+    srcs = [
+        "google/rpc/status.proto",
+    ],
+    include = ".",
+    default_runtime = "@com_google_protobuf//:protobuf_python",
+    protoc = "@com_google_protobuf//:protoc",
+    visibility = ["//visibility:public"],
+    deps = ["@com_google_protobuf//:protobuf_python"],
 )
-""",
-    )
+"""
 
-    http_archive(
-        name = "com_github_gogo_protobuf",
-        sha256 = GOGOPROTO_SHA256,
-        strip_prefix = "protobuf-" + GOGOPROTO_RELEASE,
-        url = "https://github.com/gogo/protobuf/archive/v" + GOGOPROTO_RELEASE + ".tar.gz",
-        build_file_content = """
+GOGOPROTO_BUILD_CONTENT = """
```

Comment:
Should this be at the top next to the other build content defs? Or should they all be next to their consuming sites? Either is good, as long as we are consistent.